metabase/metabase
# Allow action success message to be set

Epic https://github.com/metabase/metabase/issues/28211

Changes:
- Changes `InlineActionSettings` to use standard components for forms fields
- Adds an input to set `successMessage` for an action
- Changes `PublicAction` to use this message

How to verify:
- Create an action, go its settings, and change the default message to something else
- Make the action public, open it and submit. You should see your defined message

<img width="1495" alt="Screenshot 2023-02-10 at 16 36 58" src="https://user-images.githubusercontent.com/8542534/218118395-07f745a5-8ea0-48dc-a523-3acbcd1ad0a9.png">

<img width="1114" alt="Screenshot 2023-02-10 at 16 36 44" src="https://user-images.githubusercontent.com/8542534/218118500-655ee152-7c45-4184-abf9-c4d0b113815e.png">


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/28214)
